nginx-certbot/startup.sh

```diff
@@ -16,7 +16,8 @@ fi
 if [ "$TEST_CERT" = 'true' ] ; then
   cmd=$cmd" --test-cert"
 fi
+service nginx start
 $cmd
-service nginx stop
+pkill -9 nginx
```

Comment:
pkill can lead to zombie process ?

Reply:
It would be better to know why it fails with this timeout errors?

Reply:
@abom it's a problem on some nodes :((